apps/sap-commerce-cloud/frontend/src/AppConfig/AppConfig.tsx

```diff
@@ -39,18 +39,39 @@ interface Props {
   description: string;
 }
 
-export default function AppConfig(props: Props) {
+export default function AppConfig({
+  sdk,
```

Comment:
None of this is related to a fix I just like having the props destructured this way and noticed it